uva/branding.md

```diff
@@ -0,0 +1,22 @@
+# Branding for SGD at UVA
+
+Some general guidelines follow:
+* SGD at UVA must always be presend as SGD at UVA or Student Game Developers at the University of Virginia or some derivative.
+* SGD at UVA may **not** be refered to as UVA SGD for any official purposes. In order to do this, SGD at UVA will have to achieve special status with the university.
+* All uses of any SGD at UVA branding must support one of the following logos in a recognizable context.
+
+
+##### Square Logos
+* ![black on alpha](./Logo(black).png)
```

Comment:
Bulleted lists may not work well with images, it would probably make more sense to do:
```
# header

![image1](url)

![image2](url)
```